src/js/index.js

```diff
@@ -1,27 +1,20 @@
 import {$$} from 'n-ui-foundations';
-import {products as getUserProducts} from 'next-session-client';
 import getUserStatus from './get-user-status';
 import {init as initDataStore} from './data-store';
 import {init as initIconify} from './iconify';
 import {init as initDownloadModal} from './modal-download';
 import {init as initNavigation} from './navigation';
+import { getSyndicationAccess } from './user-access';
+import { SYNDICATION_ACCESS } from './config';
 
-async function checkIfUserIsSyndicationCustomer () {
-	const SYNDICATION_PRODUCT_CODE = 'S1';
-	const response = await getUserProducts().catch(err => err);
-
-	return response && response.products
-		? response.products.includes(SYNDICATION_PRODUCT_CODE)
-		: false;
-}
-
-async function init (flags) {
+export async function init (flags) {
 	if (!flags.get('syndication')) {
 		return;
 	}
 
-	const userIsSyndicationCustomer = await checkIfUserIsSyndicationCustomer();
-	if (!userIsSyndicationCustomer) {
+	const syndicationAccess = await getSyndicationAccess();
+
+	if (syndicationAccess.length === -1 || !syndicationAccess.includes(SYNDICATION_ACCESS.STANDARD)) {
```

Comment:
Why the comparison to `-1`? As far as I can tell, we can get either an empty array or an array of stuff... Were you thinking of `findIndex`?

Reply:
oops good spot.